src/main/java/com/github/martapietka/ppm2pbm/ByteReader.java

```diff
@@ -7,19 +7,18 @@ public class ByteReader {
 
     public static int convertBytesToInt(InputStream inputStream) throws IOException {
 
-        byte[] array = inputStream.readAllBytes();
-
         StringBuilder sb = new StringBuilder();
 
-        for (int a=0; a < array.length; a++) {
-            if (array[a] <= 0x1F) {
+        int number;
```

Comment:
Zastanawiam się czy bardziej znacząca nazwa, typu `byteValue` by nie była na miejscu? Wtedy będzie bardziej klarowne co to takiego.